.husky/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Also make sure you made this executable when you push from windows, as it's not by default

Reply:
I used Gitpod to push this file, here's the output of running `ls -l` on the filename:

```bash
gitpod /workspace/renovatebot.github.io $ ls -l .husky/pre-commit 
-rwxr-xr-x 1 gitpod gitpod 544 Jan 24 16:31 .husky/pre-commit
```